crates/iota/src/name_commands.rs

```diff
@@ -1030,7 +1031,8 @@ impl AuctionCommand {
 #[command(rename_all = "kebab-case")]
 pub enum SubnameCommand {
     /// Register a new leaf subname, which will NOT create an NFT but instead
-    /// is managed by its parent NFT. Therefore it won't be listed by the CLI.
+    /// is managed by its parent NFT. Note that leave subnames are not listed by
```

Comment:
I wouldn't even mention it here

Reply:
hmm, I actually would, because it might make the user reconsider their decision.

Reply:
sorry, but I disagree ... it's making transparent to the user that there's catch with using leaf subnames.
@Thoralf-M @

Reply:
I think the point is not to warn the user but to inform them of a limitation in one particular command. The decision to use leafs or nodes should be based on reading the docs, not the register command help text

Reply:
I don't disagree with this but I don't see the harm in having it. If they haven't yet read the docs at least they are aware 🤷🏻

Reply:
the flaw in your argument is the SHOULD :trollface:

Reply:
Alright, if we're all happy, then I remove it. I don't actually mind that hard.

Reply:
I won't push too hard for it so feel free to merge, but it doesn't make any sense to me. We can avoid inevitable confusion with a simple mention, zero reason not to do it.